app/.server/api/middlewares.test.ts

```
@@ -0,0 +1,413 @@
+
+const validAccessToken =
+  'eyJhbGciOiJIUzUxMiJ9.eyJwYXlsb2FkIjp7InNlc3Npb25JZCI6IjM5IiwidXNlciI6eyJpZCI6IjIxIiwiZW1haWwiOiJleGFtcGxlQGdtYWlsLmNvbSJ9fSwidHlwZSI6ImFjY2Vzc190b2tlbiIsImF6cCI6Imh0dHA6Ly9sb2NhbGhvc3Q6NTE3MyIsImlhdCI6MTc0MTM1ODgxMSwiZXhwIjo0ODY1NTYxMjExfQ.jZ91vx6eX6wxMg6sCsbJvKlY4DoYoQJ5Vk26OIwYPBgunInBVhyjkwqSvFCyz_e5Vxm4vn_N3HcVpOpRopa-OA';
+// const validRefreshToken = 'eyJhbGciOiJIUzUxMiJ9.eyJwYXlsb2FkIjp7InNlc3Npb25JZCI6IjM5IiwidXNlciI6eyJpZCI6IjIxIiwiZW1haWwiOiJleGFtcGxlQGdtYWlsLmNvbSJ9fSwidHlwZSI6InJlZnJlc2hfdG9rZW4iLCJhenAiOiJodHRwOi8vbG9jYWxob3N0OjUxNzMiLCJpYXQiOjE3NDEzNTg4NTgsImV4cCI6NDg2NTU2MTI1OH0.VLSnEvdTWscsmhhFUgP1XLVKEF7rpvJsXbMVrB_PEQQDVO71VfLcyNeFZhJIZv979oSKQENurSz2LBtLuceM8g';
+
+describe('getRequestData()', () => {
+  it('should return token and tokenPayload', async () => {
+    // implement tests for getRequestData() at `./middlewares.ts`
+    const { token, tokenPayload } = await getRequestData({
+      request: new Request('https://example.com', {
+        headers: new Headers({
+          Authorization: 'Bearer ' + validAccessToken,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)